src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceArea/ResourceArea.html

```diff
@@ -11,11 +11,7 @@
     </adh-space>
 
     <adh-space data-key="user">
-        <div data-adh-moving-columns="" class="moving-columns">
-            <adh-moving-column>
-                <adh-user-detail-column></adh-user-detail-column>
-            </adh-moving-column>
-        </div>
+        <adh-user-workbench></adh-user-workbench>
```

Comment:
@xi: what was your motivation for calling it a workbench? The analogy with the processes' workbenches, right?